app/modules/events/card.tsx

```diff
@@ -0,0 +1,15 @@
+import type { Event } from '@prisma/client';
+
+export function EventCard({ name, date, description, imgUrl, imgAlt, location }: Event) {
+  return (
+    <div className="grid grid-cols-3 gap-8 p-4 bg-primary text-white rounded-xl">
+      <div>{imgUrl && imgAlt && <img src={imgUrl} alt={imgAlt}></img>}</div>
```

Comment:
It looks like there will be an empty `div` here when there's no `imgUrl`. Per our discussion, this is in order for the grid layout to work properly.